pandas/compat/pickle_compat.py

```diff
@@ -104,7 +104,11 @@ def load_reduce(self):
     ('pandas.tseries.index', 'DatetimeIndex'):
         ('pandas.core.indexes.datetimes', 'DatetimeIndex'),
     ('pandas.tseries.period', 'PeriodIndex'):
-        ('pandas.core.indexes.period', 'PeriodIndex')
+        ('pandas.core.indexes.period', 'PeriodIndex'),
+
+    # 18543 moving period
```

Comment:
is there a reason you are changing this? this is a private import and should not be needed

Reply:
I saw `('pandas._period', 'Period'): ('pandas._libs.period', 'Period'),` on Line 77, and was under the impression I needed to add something for this too.

Although I now realise that even if I did need this code, it would be wrong cause I'd need to modify the value on L77 too.

Would you like me to just remove this new line? and should I modify L77 too?

Reply:
hmm ok
why don’t u put the line right under the other one then

Reply:
Good point, they seemed to be 'batched' by issue, but it'd be more readable if it was under the original, will move!